frontend/packages/ceph-storage-plugin/integration-tests-cypress/support/index.ts

```diff
@@ -26,6 +26,7 @@ Cypress.Commands.add('install', (mode: 'Internal' | 'Attached' = 'Internal', enc
       cy.log('Subscribe to OCS Operator');
       cy.byLegacyTestID('operator-install-btn').click({ force: true });
       cy.byTestID('Operator recommended namespace:-radio-input').should('be.checked');
+      cy.byTestID('enable-monitoring').click();
```

Comment:
is it to support 4.6?

Reply:
can you put a comment, if yes.

Reply:
yes. We need to remove it for 4.7

Reply:
it's required all the time, but we can remove it once 4.7 comes.